### PR TITLE
restarts celery on deploy

### PR DESCRIPTION
**Pull request by owocki**
##### Description

restarts celery on deploy

##### Refers/Fixes

deploy channel

##### Testing

still working out kinks.  see deploy channel

### CODE REVIEW

File: scripts/deploy.bash [hunk 1]
```diff
@@ -84,7 +84,8 @@ if ! [ "$JOBS_NODE" ]; then
     sudo kill -1 $pid
     sleep 0.5
     done
-
```
Comment (owocki) [inline scripts/deploy.bash]: is there another env var w3e should use to detect if were on a celery node?

Comment (androolloyd) [reply scripts/deploy.bash]: We should add one to indicate its a celery box, rather than checking for a file system.

Comment (owocki) [reply scripts/deploy.bash]: is that something you can do? im not sure the correct way to do so.. i just submitted this PR bc celery wasnt working in production the other day..

**Comment by androolloyd** [comment]
I'll add `export CELERY_NODE=1` to /etc/profile on the celery box to match this change.

**Comment by danlipert** [comment]
@androolloyd this ready to go?

**Pull request merged**